### ISSUE TITLE
yum update jubatus from v0.6.x-0.7.x to v0.8.x+ may fail if apache-log4cxx-devel is installed

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#1061]
In Jubatus 0.8.0 we renamed our `apache-log4cxx` RPM package to `log4cxx` to support RHEL 7 (#1004).
In the new SPEC file, obsoleteness of `apache-log4cxx` (by `log4cxx`) is declared, however obsoleteness of `apache-log4cxx-devel` (by `log4cxx-devel`) is not declared.

This causes RPM update failure in the following scenario:

1. (After Jubatus 0.7.2 release) `yum install apache-log4cxx-devel`
1. (After Jubatus 0.8.0 release) `yum update`

The error message is:

```
Error: Package: apache-log4cxx-devel-0.10.0-1.el6.x86_64 (@jubatus)
           Requires: apache-log4cxx = 0.10.0-1.el6
           Removing: apache-log4cxx-0.10.0-1.el6.x86_64 (@jubatus)
               apache-log4cxx = 0.10.0-1.el6
           Obsoleted By: log4cxx-0.10.0-1.el6.x86_64 (jubatus)
               Not found
```

We should explicitly obsolete `apache-log4cxx-devel` by `log4cxx-devel` package.

**Comment by kmaehashi** [comment]
If you face this error, you can workaround the problem by reinstalling apache-log4cxx-devel package.

```
yum remove apache-log4cxx-devel
yum install log4cxx-devel
```

This issue only affects to users installing apache-log4cxx-devel, which is only needed to build Jubatus from source or his own services.

**Comment by kmaehashi** [comment]
In addition, if you're using EPEL on CentOS 6.x, `yum update` from v0.6.0-v0.7.x to v0.8.0+ may fail with an error that says `apache-log4cxx` conflicts with `log4cxx`.  In that case you can workaround by:

```
yum remove apache-log4cxx
yum clean all
yum install log4cxx
```

**Comment by rimms** [comment]
:+1:

**Comment by kmaehashi** [comment]
Fixed via #1062
